@types/node@13 breaks `instanceof Module` checks

This fails to compile under `@types/node@13.1.0`, and it successfully compiles under `@types/node@12.12.22`:

```ts
import Module = require('module');
console.log(module instanceof Module);
```

The compile error under `@types/node@13.1.0` is:

```
$ tsc -v
Version 3.7.4
$ tsc
src/index.ts:2:31 - error TS2359: The right-hand side of an 'instanceof' expression must be of type 'any' or of a type assignable to the 'Function' interface type.

2 console.log(module instanceof Module);
                                ~~~~~~
```

The instanceof does work at runtime under the newest Node 13:

```
$ node -v
v13.5.0
$ node --eval "const Module = require('module'); console.log(module instanceof Module);"
true
```

And `tsc` can compile it with `@types/node@12`:

```
$ npm i --save-dev @types/node@12

+ @types/node@12.12.22
updated 1 package and audited 323 packages in 2.335s
found 0 vulnerabilities
$ tsc
$ echo $?
0
$ node build/src/index.js
true
```


- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @SimonSchick @andrewbranch

--- a/types/node/module.d.ts
+++ b/types/node/module.d.ts
@@ -16,4 +16,5 @@ declare module "module" {
 
         constructor(id: string, parent?: Module);
     }
+    export = Module;
 }

--- a/types/node/test/module.ts
+++ b/types/node/test/module.ts
@@ -1,4 +1,4 @@
-import { Module } from 'module';
+import Module = require('module');
 import { URL } from 'url';
 require.extensions[".ts"] = () => "";
 
@@ -10,6 +10,7 @@ const m2: Module = new Module("moduleId");
 const b: string[] = Module.builtinModules;
 let paths: string[] = [];
 paths = m1.paths;
+m1 instanceof Module;
 
 Module.createRequireFromPath('./test')('test');
 

